frontend/src/metabase/api/tags/utils.ts

```diff
@@ -326,3 +326,9 @@ export function provideTimelineTags(
 export function provideUserTags(user: UserInfo): TagDescription<TagType>[] {
   return [idTag("user", user.id)];
 }
+
+export function provideUserListTags(
```

Comment:
These functions should be ordered alphabetically, i.e. this one should be defined before `provideUserTags`.